exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/OperatorWrapper.java

```diff
@@ -38,7 +39,16 @@
  * Wrapper class for profiles of ALL operator instances of the same operator type within a major fragment.
  */
 public class OperatorWrapper {
+  //private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(OperatorWrapper.class);
```

Comment:
If logger is not needed, remote it.

Reply:
Ok. I see a lot of other places where it is commented out. I'll uncomment it with an unused tag for the compler to ignore. Comes very handy in debugging when making changes